css/styles.css

```diff
@@ -0,0 +1,18 @@
+* {
+  box-sizing: border-box;
+  font-family: "Courier New", Courier, monospace;
+}
+
+h1 {
+  text-transform: uppercase;
+}
+
+header {
+  background: #030007;
+  color: #ffffff;
+  height: 100px;
+  width: 100%;
+  position: fixed;
+  margin: 0 0 0 0;
```

Comment:
kurzform:

margin 0;

allerdings würde ich empfehlen ein reset oben bei dem * selektor zu machen

* {
margin: 0;
padding: 0;

Reply:
When I set the rest the class of the question card will flip. May We can look into it together?